### DISCUSSION TITLE
Thinking of dropping SQLite support (need feedback)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#633] [Polls]
In a nutshell I'm thinking of dropping support for sqlite databases and instead forcing everyone into  MySQL/MariaDB/Postgresql. Really looking for community feedback here as this will likely be a breaking change.

### Motivation for this change...
1. Would drastically simplify the "start up" and "install" processes
2. Would make the container stateless and remove having to mount local storage
3. Would make introducing backups soooo much easier

**Comment by alexdelprete** [comment]
Some italian guy I know suggested this quite some time ago. :)

Anyway, glad you came to this conclusion. I support it 100%.

**Reply by alexjustesen** [reply]
Sometimes I'm just slow to the conclusion. Now its a problem so now I want to get rid of it.

**Reply by alexdelprete** [reply]
now it has become a problem...but I was already worried about it since the beginning, having experienced it in other projects.

RPi users won't be happy...keep that in mind. I think 85% of the performance issues are for RPi users...but not just because of sqlite...I think php-fpm (another advice I gave you in that period) is another thing you should get rid of. I'll patiently wait on the border of the river for you to get tired of that one too...:)

**Comment by garanda21** [comment]
I'm 100% in agreement. I'm using it, but my UI navigation is always extremely slow compared to MySQL. 

The question is, would it be an easy way to migrate data for non-advanced users?

**Reply by alexjustesen** [reply]
I'm working on a backup/restore feature currently which will facilitate this. Been swamped with client work lately so this project hasn't gotten the love it should. Hopefully more free time coming soon
